src/main/java/org/jabref/logic/importer/fileformat/EndnoteXmlImporter.java

```diff
@@ -125,366 +160,300 @@ public ParserResult importDatabase(BufferedReader input) throws IOException {
+                        parsePeriodical(reader, entry, elementName);
                     }
                     case "keywords" -> {
-                        handleKeywordsList(reader, keywordList, elementName);
-                    }
-                    case "abstract" -> {
-                        parseStyleContent(reader, fields, StandardField.ABSTRACT, elementName);
-                    }
-                    case "isbn" -> {
-                        parseStyleContent(reader, fields, StandardField.ISBN, elementName);
+                        parseKeywords(reader, entry);
                     }
-                    case "electronic-resource-num" -> {
-                        parseStyleContent(reader, fields, StandardField.DOI, elementName);
+                    case "urls" -> {
+                        parseUrls(reader, entry);
                     }
-                    case "publisher" -> {
-                        parseStyleContent(reader, fields, StandardField.PUBLISHER, elementName);
+                    case "dates" -> {
+                        parseDates(reader, entry);
                     }
-                    case "label" -> {
-                        parseStyleContent(reader, fields, new UnknownField("endnote-label"), elementName);
+                    // TODO: Left for future work -- test files need to be adpated
+                    // case "accession-num" -> {
+                    //    String accessionNumber = parseElementContent(reader, "accession-num");
+                    //    entry.setField(new UnknownField("accession-num"), accessionNumber);
+                    // }
+                    default -> {
+                        Field field = FIELD_MAPPING.entrySet().stream()
+                                                   .filter(e -> e.getValue().equals(elementName))
+                                                   .map(Map.Entry::getKey)
+                                                   .findFirst()
+                                                   .orElse(null);
+                        if (field != null) {
+                            String value = parseElementContent(reader, elementName);
+                            entry.setField(field, value);
+                        }
                     }
                 }
             }
-            if (isEndXMLEvent(reader) && reader.getName().getLocalPart().equals(startElement)) {
-                break;
-            }
         }
 
-        BibEntry entry = new BibEntry(entryType);
-        entry.putKeywords(keywordList, preferences.bibEntryPreferences().getKeywordSeparator());
-
-        entry.setField(fields);
-        entry.setFiles(linkedFiles);
-        bibItems.add(entry);
-    }
+        // Cleanup: Remove alt-title if it matches the journal
+        String journalOrBooktitle = entry.getField(StandardField.JOURNAL).or(() -> entry.getField(StandardField.BOOKTITLE)).orElse("");
+        if (entry.hasField(new UnknownField("alt-title"))) {
```

Comment:
exctract `new UnknownField("alt-title")` to class constant `FIELD_ALT_TITLE`. You can use intellij refactoring extract field - <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>F</kbd>.

Reply:
Done.